protocol.md

```diff
@@ -0,0 +1,338 @@
+Status Secure And Decentralized Messaging Protocol
```

Comment:
See SIP-1 PR for rough draft, can add metadata and link to README if you want to merge, even while we refine more precise format